src/components/Nav/Nav.js

```diff
@@ -1,7 +1,87 @@
-import React from 'react';
+import { AiOutlineMenu } from 'react-icons/ai';
+import { CgSearch } from 'react-icons/cg';
+import styled from 'styled-components';
+import React, { useState } from 'react';
+import SideDrawer from './SideDrawer';
+import GlobalFonts from '../../fonts/fonts';
 
 const Nav = () => {
-  return <div>Nav</div>;
+  const [navDrawer, setNavDrawer] = useState(false);
+
+  const handleNavDrawer = () => {
+    setNavDrawer(!navDrawer);
```

Comment:
- 곧바로 값을 바꿔줘도 되지만 함수를 사용해서 이전 값을 통한 변경이 좀 더 안전합니다.

Reply:
멘토님 말씀하신 부분이 어떤건지 감이 안옵니다🥲 제가 뭘 변경하면 좋은걸까요?

Reply:
> 멘토님 말씀하신 부분이 어떤건지 감이 안옵니다🥲 제가 뭘 변경하면 좋은걸까요?

- 이런 식으로 이전 값을 받아서 진행하시는게 좀 더 안전합니다.
```suggestion
    setNavDrawer(prev => !prev);
```

Reply:
감사합니다. 수정 완료하였습니다 😄